compatibility/mysql/csharp/MySQLTest.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using MySql.Data.MySqlClient;
+using System.IO;
+
+public class MySQLTest
+{
+    public class Tests
+    {
+        private MySqlConnection conn;
+        private MySqlCommand cmd;
+        private List<Test> tests = new List<Test>();
+
+        public void Connect(string ip, int port, string user, string password)
+        {
+            try
+            {
+                string connectionString = $"Server={ip};Port={port};User Id={user};Password={password};";
+                conn = new MySqlConnection(connectionString);
+                conn.Open();
+                cmd = conn.CreateCommand();
+                cmd.CommandType = CommandType.Text;
+            }
+            catch (MySqlException e)
+            {
+                throw new Exception(e.Message);
```

Comment:
Please add necessary prefix for all `Exception`. For example, the `Exception` here can be: 
```cs
                throw new Exception($"Failed to connect to MySQL server: {ex.Message}", ex);

```